server/graphql_schemas/event/schema.py

```diff
@@ -1,65 +1,107 @@
 import graphene
 
-from graphene import relay, InputObjectType, ObjectType
+from graphene import relay, ObjectType
 from graphql_relay.node.node import from_global_id
 from graphene_django.filter import DjangoFilterConnectionField
 from graphene_django.types import DjangoObjectType
+from graphql import GraphQLError
 
-from api.models import Event, Category, Attend, GoogleUser
+from api.models import Event, Category, GoogleUser
 
 
 class EventNode(DjangoObjectType):
-  class Meta:
-    model = Event
-    filter_fields = {}
-    interfaces = (relay.Node,)
+    class Meta:
+        model = Event
+        filter_fields = {}
+        interfaces = (relay.Node,)
 
 
 class CreateEvent(relay.ClientIDMutation):
     class Input:
-      title = graphene.String(required=True)
-      description = graphene.String(required=True)
-      venue = graphene.String(required=True)
-      date = graphene.String(required=True)
-      time = graphene.String(required=False)
-      featured_image = graphene.String(required=False)
-      social_event_id = graphene.Int(required=False)
+        title = graphene.String(required=True)
+        description = graphene.String(required=True)
+        venue = graphene.String(required=True)
+        date = graphene.String(required=True)
+        time = graphene.String(required=False)
+        featured_image = graphene.String(required=False)
+        social_event_id = graphene.String(required=False)
 
     new_event = graphene.Field(EventNode)
 
     @classmethod
     def mutate_and_get_payload(cls, root, info, **input):
         social_event_id = input.get('social_event_id')
-        social_event = Category.objects.get(id=int(social_event_id))
+        social_event = Category.objects.get(
+            id=from_global_id(social_event_id)[1])
         user = info.context.user
```

Comment:
@parseendavid Please why line 36? because `user` is not used due change in line 37.